### PR TITLE
Feat/Car Damage Assessment

### PR DESCRIPTION
**Pull request by rizkybaihaqy**
## Type of changes

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Enhancement
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the current behaviour?

there was nothing 

## What is the new behavior?

here are some features I want you to see: 
- Users can select one of the four predefined images per side of the car (front, left, right, rear)
- After the user selects a car image, there will be a highlight of the selected image
- Each side of the car can only be selected one image, if the user has selected an image and then chooses again, then the next selected image will be selected
- If the user does not select one in each part of the car, an error message will appear along with highlighting the side of the car that has not been selected and the user cannot continue their adventure.
- The error message and the highlight will disappear if the user selects a car image on that side
- there is loading while the request is being processed
- If the request is successful, the condition prediction and score from the picture will be displayed both per side of the car and also overall from all four sides of the car
- if the request fails it will just display the message "oops, something went wrong" regardless of the error condition
- whether it succeeds or fails, a feedback form will be displayed
- image is responsive to screen size

## Checklist

- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have filled the `[Unreleased]` [Changelog](https://github.com/nodefluxio/weber/blob/main/CHANGELOG.md

## Security Pentester API Checklist
- [ ] IDOR
- [ ] Malicious Script Injection
- [ ] SQL Injection

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

Don't be intimidated by the 18 files changed, 12 of which are just adding image files

some screenshots:
image
image
image
demo
image
*went
image
image

### CODE REVIEW

**Review by sonadztux — APPROVED** [review]
works well on my maching. gg!
Peek 2021-12-02 20-04

File: frontend/app/types/elements.ts [hunk 1]
```diff
@@ -101,3 +101,40 @@ export type PaymentAccountInfo = {
...
+  left: T
+  right: T
+  rear: T
+}
+
+export type CarDamageSample = {
+  front: string[]
+  left: string[]
+  right: string[]
+  rear: string[]
```
Comment (williammfu) [inline frontend/app/types/elements.ts:116]: `CarDamageSample` kenapa engga dibuat `CarDamage<string[]>` aja?

**Review by williammfu — COMMENTED** [review]
`CarDamageSample` kenapa engga dibuat `CarDamage<string[]>` aja?

File: frontend/app/types/elements.ts (hunk 1 above)
Comment (rizkybaihaqy) [reply frontend/app/types/elements.ts:116]: bener juga, okok gw ganti

Comment (rizkybaihaqy) [reply frontend/app/types/elements.ts]: re review 2b7535d

Comment (williammfu) [reply frontend/app/types/elements.ts]: resolved at 9670e46

**Review by williammfu — APPROVED** [review]
Approved untuk tidur yang nyenyak

**Review by gregoriusjimmy — COMMENTED** [review]
Functionality worked, cuman dari styling aja feedback form gepeng karena style parent containernya 
image
sama gambar yang disuruh milih cursornya belum pointer
image

**Comment by rizkybaihaqy** [comment]
yang kata lu gepeng abis UAT deh, soalnya gw ga tau sebelah mana gepengnya

**Pull request merged**